packages/edit-site/src/components/template-details/index.js

```diff
@@ -2,12 +2,17 @@
  * WordPress dependencies
  */
 import { __, sprintf } from '@wordpress/i18n';
-import { Button, __experimentalText as Text } from '@wordpress/components';
+import {
+	Button,
+	MenuItem,
+	__experimentalText as Text,
+} from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
+import isTemplateRevertable from '../../utils/is-template-revertable';
```

Comment:
Do you think there's value in having this separate, or would  it be  better to void the indirection and just compute the const in the component here?

Reply:
The logic is simple enough, true, but given it derives from API details, I'd rather keep them contained in a function, than expose them in several parts of the code.